examples/planar_vector_tiles.html

```diff
@@ -0,0 +1,94 @@
+<html>
+    <head>
+        <title>Itowns - Planar + TMS + Vector Tile</title>
+
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="css/example.css">
+        <link rel="stylesheet" type="text/css" href="css/loading_screen.css">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <script src="../dist/itowns.js"></script>
+        <script src="js/loading_screen.js"></script>
+        <script src="js/VectorTileStyleSupport.js"></script>
+        <script>
+            // # Planar view with one single layer of vector tile
+
+            // Define geographic extent: CRS, min/max X, min/max Y
+            var extent = new itowns.Extent(
+                'EPSG:3857',
+                -20037508.342789244, 20037508.342789244,
+                -20037508.342789255, 20037508.342789244);
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            // Instanciate PlanarView
+            var view = new itowns.PlanarView(viewerDiv, extent, { maxSubdivisionLevel: 20 });
+
+            // eslint-disable-next-line no-new
+            new itowns.PlanarControls(view, {
+                // We do not want the user to zoom out too much
+                maxAltitude: 40000000,
+                // We want to keep the rotation disabled, to only have a view from the top
+                enableRotation: false,
+                // Faster zoom in/out speed
+                zoomInFactor: 0.5,
+                zoomOutFactor: 0.5,
+                // Don't zoom too much on smart zoom
+                smartZoomHeightMax: 100000,
+            });
+
+            // Turn in the right angle
+            view.camera.camera3D.rotateZ(-Math.PI / 2);
+            setupLoadingScreen(viewerDiv, view);
+
+            var count = 0;
+
+            // Add a vector tile layer
+            itowns.Fetcher.json('https://raw.githubusercontent.com/Oslandia/postile-openmaptiles/master/style.json').then(function (style) {
+                var supportedLayers = [];
+
+                view.tileLayer.noTextureColor =
+                new itowns.THREE.Color(style['layers'][0]['paint']['background-color']);
+
+                style.layers.forEach(function(layer) {
+                    if (layer.type === 'fill' || layer.type === 'line') {
+                        supportedLayers.push(layer);
+                    }
+                });
+
+                view.addLayer({
+                    type: 'color',
+                    protocol: 'xyz',
```

Comment:
I dunno, I think it is too much a hassle, since this (declaration of layer type, protocol etc) should be more generic in the future ? And in our example we only have the `xyz` protocol, but here at IGN we have internal example with the `tms` protocol.

Reply:
You didn't reply here?

Reply:
Sorry ! Yes we could but that means the introduction of a new provider if I'm not mistaken. And that doesn't make a lot of sense to me. We seem to both agree that the current `3dtiles` semantic is not the most appropriate, and that it should change, and I see this proposition for `vector-tiles` as doing a similar error like what we did for `3dtiles`.

IMHO we already have a lot of ongoing and oncoming PR refactoring the layers, providers, sources, parsers and other to take care of this later, WDYT ?

Reply:
Right, let's forget about the `vector-tiles` protocol for now.

Reply:
Thanks !